DCA/data/nj.py

```diff
@@ -0,0 +1,198 @@
+import random
```

Comment:
Let's upload the authors' implementation as a separate request, so that it is clear what to review